libs/common-const/src/gnosis_chain/hack.ts

```diff
@@ -2,7 +2,7 @@
 // + logic for chainId switch to/from xDAI
 import { SupportedChainId } from '@cowprotocol/cow-sdk'
 
-import { XDAI_SYMBOL } from './constants'
+import { XDAI_SYMBOL } from '../tokens'
```

Comment:
I found strange that `getChainCurrencySymbols` is inside a folder called `gnosis_chain` but accepts any chain as a parameter

Reply:
Indeed! Considering the comment in the file beginning it must be an artifact from Uniswap era.
To avoid the scope blowing I prefer to fix it in the another iteration